src/ui/public/vis/vis.js

```diff
@@ -12,13 +12,12 @@ import _ from 'lodash';
 import AggTypesIndexProvider from 'ui/agg_types/index';
 import RegistryVisTypesProvider from 'ui/registry/vis_types';
 import VisAggConfigsProvider from 'ui/vis/agg_configs';
-import PersistedStateProvider from 'ui/persisted_state/persisted_state';
+import { PersistedState } from 'ui/persisted_state';
```

Comment:
Since this is an angular context we need to use the angular version of `PersistedState`